HelloWorld_ST_Sensors/main.cpp

```diff
@@ -152,16 +143,14 @@ int main() {
                 spec_co_gas_conc, spec_co_temp, spec_co_rel_humidity);
 
     // GPS
-    if (is_gps_recv == true) {
-      debugPrintf("AP6H GPS:                        Latitude(deg): %f | "
-                  "Longitude(deg): %f\r\n",
-                  gps_coordinates_data.latitude,
-                  gps_coordinates_data.longitude);
-      is_gps_recv = false;
-    }
+    gps::gps_coordinates_s &gps_coordinates_data =
+        gps_obj.get_gps_coordinates();
+    debugPrintf("AP6H GPS:                        Latitude(deg): %f | "
+                "Longitude(deg): %f\r\n",
+                gps_coordinates_data.latitude, gps_coordinates_data.longitude);
 
     // Heartbeat Sensor
-    const heartbeat::bioData hb_data = heartbeat::get_hb_data();
+    const heartbeat::bioData hb_data = hb_obj.get_hb_data();
```

Comment:
const heartbeat::bioData `&` hb_data = hb_obj.get_hb_data();

Reply:
Updated in new PR.